src/tests/functional/shared_test_classes/src/base/ov_subgraph.cpp

```diff
@@ -89,7 +92,11 @@ void SubgraphBaseTest::run() {
         if (status != LayerTestsUtils::PassRate::Statuses::PASSED) {
             GTEST_FATAL_FAILURE_(errorMessage.c_str());
         }
-    } else {
+    } else if (jmpRes == CommonTestUtils::JMP_STATUS::anyError) {
+        summary.updateOPsStats(function, LayerTestsUtils::PassRate::Statuses::CRASHED);
```

Comment:
Do not forget about this [code](https://github.com/openvinotoolkit/openvino/blob/5e023ebdd9568a59f7b93337c9e8ed9159d555cc/src/tests/functional/shared_test_classes/src/base/ov_subgraph.cpp#L51)